### ISSUE TITLE
Consider out-of-band domain migrations to fix heuristic errors

### ISSUE THREAD

**Issue by cowlicks** [EFForg/privacybadger#1645]
It would be nice to be able to "unblock" domains that were erroneously blocked by privacy badger. I'm imagining something like a migration, except it can be delivered outside of privacy badger updates, like the yellowlist, or DNT hashes.

We have 2 kinds of migrations:
* a kind where we change the schema of our storage
* a kind where we unblock mistakenly blocked domains

This would address the second kind. We'd be able to more quickly deliver fixes to users, to give them a better experience. Some times where this would be useful:
* When we've broken stuff by having a bad heuristic or some other bug.
     - Like when we broke a bunch of sites because we remove some entries from the yellow list. We could have sent fixes without doing a release.
     - unblocking domains which were blocked by __cfduid cookies
* when we are unable to reproduce the tracking that cause something to be blocked. So we need to unblock it. See the [unable to reproduce issue label](https://github.com/EFForg/privacybadger/issues?q=is%3Aopen+is%3Aissue+label%3A%22unable+to+reproduce%22).
     - youtube is a good example of this right now. We would be able to try try unblocking domains related to youtube. This might help, and improve the user's experience even though we don't know what is happening.
     - #1153 we've spent a lot of energy investigating it. A good stopgap measure would be unblocking it with a simple migration. Instead of spending more time searching for the cause of the tracking.

This would also be nice because our migration system is pretty clunky. Removing domain unblocking from it would make working with it much simpler, and therefor less error prone.

**Comment by ghostwords** [comment]
Related to #795 in the sense it's another remote update feature. This one might be pretty different though. What should these updates look like? Would there be a local "definitions" file, or should this be remote-only?

**Comment by ghostwords** [comment]
Closing as a "wontfix" for now; we may revisit this in the future.
